fullstory-segment-middleware/src/test/java/com/fullstorydev/fullstory_segment_middleware/SegmentSpecUtils.java

```diff
@@ -0,0 +1,363 @@
+package com.fullstorydev.fullstory_segment_middleware;
+
+import org.mockito.Mockito;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+public class SegmentSpecUtils {
```

Comment:
I would be a good idea to add a comment to the class, outlining that you are creating fake data that maps to the Segment ecommerce spec definied here: https://segment.com/docs/connections/spec/ecommerce/v2/